setup.py

```diff
@@ -1,11 +1,22 @@
 import os
 import re
+import sys
 from codecs import open
 
 from setuptools import setup, find_packages
 
 here = os.path.abspath(os.path.dirname(__file__))
 
+PY34 = sys.version_info[0:2] >= (3, 4)
+
+install_requires = [
+        'sqlalchemy>=1.2',
+        'requests',
+        'clickhouse_driver>=0.0.14'
```

Comment:
I've bumped clickhouse-driver version to 0.0.19 for ip addresses support.

Reply:
Version bumped